src/llm/summarize/prompt.py

```diff
@@ -51,7 +51,8 @@
 {related_past_episodes}
 
 [INSTRUCTION]
-Using above [RESULTS OF COMPLETED TASKS], [RELATED KNOWLEDGE], and [RELATED PAST EPISODES], answer the [GOAL].
+ - Using above [RESULTS OF COMPLETED TASKS], [RELATED KNOWLEDGE], and [RELATED PAST EPISODES], answer the [GOAL].
+ - If not possible find the answer, return just nothing: 'I don't know'.
```

Comment:
Replace line 55 with:
- If you think performing [NEXT SUBTASK] will lead to a better answer, answer "continue"
- If it's not possible to find the answer, answer "I don't know"

Reply:
What is [NEXT SUBTASK] ? We dont have it currently. Oh [NEXT POSSIBLE TASKS]? I see.